include/zephyr/bluetooth/services/hrs.h

```diff
@@ -34,6 +36,15 @@ struct bt_hrs_cb {
 	 */
 	void (*ntf_changed)(bool enabled);
 
+	/** @brief Heart rate control point write callback
+	 *
+	 * @param request control point request code
+	 *
+	 * @return int 0 on successful handling of control point request
+	 *             negative error codes
+	 */
+	int (*ctrl_point_write)(uint8_t request);
```

Comment:
Please add reference to `BT_HRS_CONTROL_POINT_RESET_REQ` so that users can relate `request` to the possible value(s)